src/index.ts

```diff
@@ -21,8 +21,16 @@ export interface PyroscopeConfig {
   configured: boolean
 }
 
-const INTERVAL = 10000
-const SAMPLERATE = 100
+// https://github.com/google/pprof-nodejs/blob/0eabf2d9a4e13456e642c41786fcb880a9119f28/ts/src/time-profiler.ts#L37-L38
+// The Interval in which samples should be collected. (in microseconds)
+// converting to microseconds
+const SAMPLING_INTERVAL =  process.env['PYROSCOPE_SAMPLE_RATE'] || 1000 // in milliseconds
+
+
+// The Duration for which a sample should be collected.
+// https://github.com/google/pprof-nodejs/blob/0eabf2d9a4e13456e642c41786fcb880a9119f28/ts/src/time-profiler.ts#L35-L36
+const SAMPLING_DURATION =  process.env['PYROSCOPE_SAMPLING_DURATION'] || 10000 // in milliseconds
```

Comment:
What do you think about renaming `PYROSCOPE_SAMPLE_RATE` to `PYROSCOPE_ SAMPLING_INTERVAL`?

Also, re names of constants – should we maybe add a unit suffix? Like:
```
SAMPLING_INTERVAL_MS
SAMPLING_DURATION_MS
```

Reply:
makes sense, totaltly!